src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -91,92 +84,137 @@ public function read(array $domains, array $locales): TranslatorBag
     {
         $domains = $domains ?: ['*'];
         $translatorBag = new TranslatorBag();
+        $responses = [];
 
         foreach ($locales as $locale) {
             foreach ($domains as $domain) {
-                $response = $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain));
+                $responses[] = [
+                    'response' => $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', rawurlencode($locale), rawurlencode($domain))),
+                    'locale' => $locale,
+                    'domain' => $domain,
```

Comment:
I would prefer:
```suggestion
                    'response' => $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated',
                    rawurlencode($locale),
                    http_build_query([
                        'filter' => $domain,
                        'status' => 'translated',
                    ])),
                    'locale' => $locale,
                    'domain' => $domain,
```

Hopefully I get it right via the UI, but the key is to use `http_build_query`